R/plot.emdi.R

```diff
@@ -269,12 +269,12 @@ plot.emdi <- function(x,
       x_lab <- label$opt_lambda["x_lab"]
     }
     if (any(is.na(likelihoods))) {
-      warning(paste0(
-        "For some lambda in the chosen range, the ",
-        "likelihood does not converge. ",
-        "For these lambdas no likelihood is plotted. ",
-        "Choose a different range to avoid this behaviour"
-      ))
+      warning(strwrap(prefix = " ", initial = "",
+                      paste0("For some lambda in the chosen range, the
+                             likelihood does not converge. For these lambdas,
+                             no likelihood is plotted. Choose a different range
+                             to avoid this behaviour"
+                             )))
```

Comment:
```suggestion
      warning(strwrap(prefix = " ", initial = "",
                      "For some lambda in the chosen range, the
                      likelihood does not converge. For these lambdas,
                      no likelihood is plotted. Choose a different range
                      to avoid this behaviour"
                      ))
```

Here and at other parts of the code, `paste0` is still used but not necessary.